### PR TITLE
2) Trimming submission ids #90

### PR DESCRIPTION
**Pull request by naazy**
#90 https://github.com/OxfordAbstracts/application/issues/2793

### CODE REVIEW

File: src/MainUpdate.elm [hunk 1]
```diff
@@ -336,6 +336,7 @@ update msg model =
                     invalidSubmissionIds =
                         newSubmissionIdsString
                             |> split ","
+                            |> List.map (\sub -> String.trim sub)
```
Comment (roryc89) [inline src/MainUpdate.elm]: no need for anonymous function here. Can be ` List.map String.trim`. Doesn't need to be changed but maybe useful for future PRs

**Comment by roryc89** [comment]
One comment for future reference but definitely not a show stopper! Liking the flood of PRs!

Assigning Nelson.

File: src/MainUpdate.elm (hunk 1 above)
Comment (naazy) [inline src/MainUpdate.elm]: Good point

Comment (roryc89) [inline src/MainUpdate.elm]: 👍 👍

**Pull request merged**